src/tracer.js

```diff
@@ -254,23 +257,26 @@ export default class Tracer {
     }
 
     /**
-     *  Makes a concrete sampling decision for the ctx span context based on information
-     *  available to it's child span.
-     *
-     *  The ctx context's deferred mask is unset after this decision is made
+     * Reapplies sampling if the parent span isn't sampled
      *
+     * @param config the upsampling config
+     * @param parentCtx the parent span context
      * @param ctx the span context
      * @param operationName the operation name for the child
      * @param tags tags to be applied by the sampler
      */
-    processDeferredSampling(ctx: SpanContext, operationName: string, tags: any) {
-        if (ctx.isDeferredSampling) {
+    handleUpsampling(config: any, parentCtx: SpanContext, ctx: SpanContext,
+                     operationName: string, tags: any) {
+        if (!config.enabled) {
```

Comment:
if config is null you can get an exception often in ES6 people initialize a default dictionary, or write config = config || {}